updating/installing-update-service.adoc

```diff
@@ -10,20 +10,22 @@ toc::[]
 // modules required to cover the user story. You can also include other
 // assemblies.
 
-For clusters with internet accessibility, Red Hat provides over-the-air updates through an OpenShift Container Platform update service as a hosted service located behind public APIs. However, for clusters in a restricted network, there’s no way to access these public APIs to obtain this information. In order to provide a similar user upgrade experience in a restricted network users can install and configure the OpenShift Update Service locally, i.e. accessible within their disconnected network, to provide over-the-air updates for your cluster and its underlying operating system as described in the following sections.
```

Comment:
I think it's important to keep `accessible within their disconnected network` which really defines what installing the service `locally` achieves.

Reply:
Got it. I added this distinction but chose "disconnected environment" to match the phrasing used in other parts of OCP docs. 

>To provide a similar upgrade experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.